packages/fluentui/react-bindings/src/hooks/useFrameFocusDispatch.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+export const useFrameFocusDispatch = (
+  enableFrameFocusDispatch: boolean,
+  $document: Document,
+  pollDuration: number = 1000,
+) => {
+  const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>();
+
+  React.useEffect(() => {
+    if (enableFrameFocusDispatch) {
+      timeoutRef.current = setInterval(() => {
+        const activeElement = $document?.activeElement;
+        if (activeElement?.tagName === 'IFRAME') {
+          const event = new CustomEvent(FUI_FRAME_EVENT, { bubbles: true });
+          activeElement.dispatchEvent(event);
+        }
+      }, pollDuration);
+    }
+    return () => {
+      clearTimeout(timeoutRef.current as ReturnType<typeof setTimeout>);
+    };
+  }, [$document?.activeElement, enableFrameFocusDispatch, pollDuration]);
```

Comment:
```suggestion
  }, [$document, enableFrameFocusDispatch, pollDuration]);
```

The dependency there is not correct as we actually have a dependency there on `$document`